### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ReneSkukies**
unify readme with unfold family

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,33 +23,96 @@ The supportet visualizations are:
...
+If you make use of theses visualizations, please cite:
 
-It was authored by `Benedikt Ehinger` and has been worked upon by students and doctoral researchers within the context of a university project (`Fachpraktikum:  Methods in Computational EEG`) at the University of Stuttgart.
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6531996.svg)](https://doi.org/10.5281/zenodo.6531996)
 
-The student names are:
-- Baumgartner, Daniel
-- Döring, Sören
-- Gärtner, Niklas
+## Contributors (alphabetically)
```
Comment (behinger) [inline README.md:107]: you are a contributor too :)

File: README.md [hunk 2]
```diff
@@ -22,33 +23,96 @@ The supportet visualizations are:
 - Topoplot series
 - Parallel Coordinates Plot
 
+<details>
+<summary>Click to see example plots</summary>
```
Comment (behinger) [inline README.md:27]: we need to update this plot @vladdez do we have a more exciting and more importantly, higher resolution plot?

We also shoudnt hide it, it is a plotting library. Maybe we can shrink it, and not show all plot types?

File: README.md [hunk 3]
```diff
@@ -22,33 +23,96 @@ The supportet visualizations are:
 - Topoplot series
```
Comment (behinger) [inline README.md:23]: Circular topoplots are missing

also the text above is suboptimal, we should remove the lines

The toolbox requires a Makie-Backend to be loaded, e.g. `using CairoMakie`
Some plotting functions require data-frames, with fields like `:estimate` or `:time`.

Or at least send them down to the quick start or something

File: README.md (hunk 1 above)
Comment (ReneSkukies) [reply README.md:107]: So far only because I updated the readme, no? Didn't think that would justify being on the list 👀

File: README.md (hunk 2 above)
Comment (ReneSkukies) [reply README.md:27]: Shrinking it would be nice, I hid it cause it was taking up a looooooot of space

**Pull request merged**